recipes/unsync/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "unsync" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/alex-sherman/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 7a799a21fa7f7564c55e7de781109390e47213e17f676e9b88211cb0f4ceef50
+
+build:
+  number: 0
```

```suggestion
  number: 0
  noarch: python
```

can this be noarch python?

---

I originally did put it as noarch... I only hesitated because because it uses async I didn't know if I should set a minimum Python version of 3.5 or 3.6 which made the linter complain... happy to remove the Python min Python version and make that change though.

---

right. IDK either. It should test the recipe on 3.6 so if you put in some decent tests, it should catch any obvious errors.